PWGCF/JCorran/DataModel/JCatalyst.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+/// \author Jasper Parkkila (jparkkil@cern.ch)
+/// \author Dong Jo Kim (djkim@jyu.fi)
+/// \since Sep 2022
+#ifndef JFLUC_CATALYST_H
+#define JFLUC_CATALYST_H
+
+namespace o2::aod
+{
+namespace particleTrack
+{
+DECLARE_SOA_INDEX_COLUMN(Collision, collision);
+DECLARE_SOA_COLUMN(Pt, pt, float);
+DECLARE_SOA_COLUMN(Eta, eta, float);
+DECLARE_SOA_COLUMN(Phi, phi, float);
+DECLARE_SOA_COLUMN(WeightNUA, weightNUA, float);
+DECLARE_SOA_COLUMN(WeightEff, weightEff, float);
```

Comment:
Please, comment your column declaration for derived tables so they can appear in the documentation

Reply:
For guidance check `Common/DataModel/Centrality.h`

Reply:
Ok, added!